src/rust/engine/src/intrinsics.rs

```diff
@@ -515,147 +517,151 @@ fn interactive_process(
-  async move {
-    let types = &context.core.types;
-    let interactive_process_result = types.interactive_process_result;
-
-    let (py_interactive_process, py_process, process_config): (Value, Value, externs::process::PyProcessConfigFromEnvironment) = Python::with_gil(|py| {
-      let py_interactive_process = (*args[0]).as_ref(py);
-      let py_process: Value = externs::getattr(py_interactive_process, "process").unwrap();
-      let process_config = (*args[1])
-        .as_ref(py)
-        .extract()
-        .unwrap();
-      (py_interactive_process.extract().unwrap(), py_process, process_config)
-    });
-    match process_config.execution_strategy {
-      ProcessExecutionStrategy::Docker(_) | ProcessExecutionStrategy::RemoteExecution(_) => Err("InteractiveProcess should not set docker_image or remote_execution".to_owned()),
-      _ => Ok(())
-    }?;
-    let mut process = ExecuteProcess::lift(&context.core.store(), py_process, process_config).await?.process;
-    let (run_in_workspace, restartable, keep_sandboxes) = Python::with_gil(|py| {
-      let py_interactive_process_obj = py_interactive_process.to_object(py);
-      let py_interactive_process = py_interactive_process_obj.as_ref(py);
-      let run_in_workspace: bool = externs::getattr(py_interactive_process, "run_in_workspace").unwrap();
-      let restartable: bool = externs::getattr(py_interactive_process, "restartable").unwrap();
-      let keep_sandboxes_value: &PyAny = externs::getattr(py_interactive_process, "keep_sandboxes").unwrap();
-      let keep_sandboxes = KeepSandboxes::from_str(externs::getattr(keep_sandboxes_value, "value").unwrap()).unwrap();
-      (run_in_workspace, restartable, keep_sandboxes)
-    });
-
-    let session = context.session;
-
-    let mut tempdir = create_sandbox(
-      context.core.executor.clone(),
-      &context.core.local_execution_root_dir,
-      "interactive process",
-      keep_sandboxes,
-    )?;
-    prepare_workdir(
-      tempdir.path().to_owned(),
-      &process,
-      process.input_digests.input_files.clone(),
-      context.core.store(),
-      context.core.executor.clone(),
-      &context.core.named_caches,
-      &context.core.immutable_inputs,
-      None,
-      None,
-    )
-    .await?;
-    apply_chroot(tempdir.path().to_str().unwrap(), &mut process);
-
-    let p = Path::new(&process.argv[0]);
-    // TODO: Deprecate this program name calculation, and recommend `{chroot}` replacement in args
-    // instead.
-    let program_name = if !run_in_workspace && p.is_relative() {
-      let mut buf = PathBuf::new();
-      buf.push(tempdir.path());
-      buf.push(p);
-      buf
-    } else {
-      p.to_path_buf()
-    };
-
-    let mut command = process::Command::new(program_name);
-    if !run_in_workspace {
-      command.current_dir(tempdir.path());
-    }
-    for arg in process.argv[1..].iter() {
-      command.arg(arg);
-    }
+  in_workunit!(
+    "interactive_process",
+    Level::Trace,
```

Comment:
Let's make this `Debug`. Because it won't have a description, it won't show up in the UI, but it _will_ be captured by default in the streaming workunits.